account_index_based_currency/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2017 Vauxoo, S.A. de C.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    'author': 'ADHOC SA,Odoo Community Association (OCA), Vauxoo',
```

Comment:
@hbto ADHOC? are you sure?

Reply:
never mind

Reply:
@yaniaular  You're right, this should be only Vauxoo.

I think this was caused by a lint that was supposed to request one of those authors to be present, but it was solved a long time ago (https://github.com/Vauxoo/pylint-conf/pull/57).

Reply:
We have ditch this PR.

A new one is to be created nothing in here is related to other partners.